document/src/main/kotlin/com/ritense/document/domain/InternalCaseStatusId.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.document.domain
+
+import com.ritense.valtimo.contract.domain.AbstractId
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+
+
+@Embeddable
+data class InternalCaseStatusId(
+    @Column(name = "case_definition_name", length = 50, columnDefinition = "VARCHAR(50)")
```

Comment:
The `case_definition_name` can be 256 long

Reply:
fixed

Reply:
As far as i can find it is only 50

Reply:
I was wrong. It should be 50 after all

Reply:
fixed